app/scripts/TilesetFinder.js

```diff
@@ -20,7 +20,12 @@ class TilesetFinder extends React.Component {
 
     // local tracks are ones that don't have a filetype associated with them
     this.localTracks = TRACKS_INFO
-      .filter(x => x.local && !x.hidden);
+      .filter(x => x.local && !x.hidden)
+      .map((x) => {
+        const y = Object.assign({}, x);
+        y.datatype = x.datatype[0];
+        return y;
```

Comment:
Could you elaborate in the PR's description what this changes? Seems to be more than just docs updates?

Reply:
In the `tracks-info.js` file, datatypes are arrays because a track can be used to view multiple different datatypes. For displaying the available track types, a tileset listed in the tileset finder needs to have just a single datatype. For local track types (e.g. top axis and left axis), the track is the tileset so we have to convert the datatype array to a single value.